flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SubtaskCheckpointCoordinatorImpl.java

```diff
@@ -328,6 +421,7 @@ private OperatorSnapshotFutures buildOperatorSnapshotFutures(
 
 	@Override
 	public void close() throws IOException {
+		asyncCheckpointRunnables.close();
```

Comment:
`try/catch` (or guava Closer or some other way to close both no matter what?)

Reply:
OK, I'll ensure both the closeables could be closed.